std_msgs/ColorRGBA[12] colors